app-admin/ruri/autobuild/prepare

```diff
@@ -0,0 +1,2 @@
+abinfo "Configuring..."
```

Comment:
No. This is now how you are supposed to call configure - if this is not a standard autotools, then don't advertise it as such.

Or, as you suggested, make a patch to make it so.

Reply:
OK.